inference/core/models/roboflow.py

```diff
@@ -401,7 +401,8 @@ def preproc_image(
             )
         elif self.resize_method == "Fit (black edges) in":
             resized = letterbox_image(
-                preprocessed_image, (self.img_size_w, self.img_size_h)
+                preprocessed_image, (self.img_size_w, self.img_size_h),
+                color=(114, 114, 114)
```

Comment:
this isn't going to work with our existing models unfortunately -- you can see the resize method is `Fit (black edges) in`. We'll need to add a new resize_method `Fit (grey edges) in` which would use this color parameter

Reply:
@NickHerrig perhaps you can give that a whirl and edit the environment.json file for deployed yolov10s

Reply:
Hey @probicheaux Yeah, I'll be able to dive into this more tomorrow! 👍

Reply:
Thanks! We have updated the PR to add a new resize_method `Fit (grey edges) in`.

Reply:
@jameslahm Thank you for your speed in adding this! 

@probicheaux, This addition looks good to me. ✅